test/src/test/java/hudson/model/JobTest.java

```diff
@@ -286,4 +291,35 @@ private static void tryConfigDotXml(JenkinsRule.WebClient wc, int status, String
         p.renameTo("different-name");
     }
 
+    @Issue("JENKINS-30502")
+    @Test public void testRenameTrimsSpace() throws Exception {
+        tryRename("myJob", " foo", "foo");
+        tryRename("myJob", "foo ", "foo");
+    }
+
+    private void tryRename(
+            String initialName,
```

Comment:
strange style, looks like you trying to fit into calculator width :)
Please place first argument after `(`